Config.cmake/test_fmi1.cmake

```diff
@@ -158,7 +153,7 @@ if(FMILIB_BUILD_BEFORE_TESTS)
         ctest_fmi_import_me_test
         ctest_fmi_import_cs_test
         ctest_fmi_import_xml_test
-        ctest_fmi_import_xml_test_empty
+        ctest_fmi_import_xml_test_brief

```

Comment:
It tests a condensed ("brief") XML. Unclear why. But it's not empty/missing at least which had me confused.